src/Service/Organization/AliasGenerator.php

```diff
@@ -10,7 +10,7 @@ public function generate(string $name): string
     {
         $text = transliterator_transliterate('Any-Latin; Latin-ASCII; [^\u001F-\u007f] remove', $name);
         $text = strtolower((string) $text);
-        $text = preg_replace('~[^-\w]+~', ' ', $text);
+        $text = preg_replace('~[^\\-\w]+~', ' ', $text);
```

Comment:
Was this added by rector? Both patterns work the same here, but this might be good to explicitly escape the `-` character, so :+1:.